librocksdb_sys/src/lib.rs

```diff
@@ -976,6 +980,46 @@ extern "C" {
                                                       props: *mut DBTablePropertiesCollection,
                                                       errptr: *mut *mut c_char);
 
+    // Get All Key Versions
+    pub fn crocksdb_keyversions_create() -> *mut DBKeyVersions;
+
+    pub fn crocksdb_keyversions_destroy(kvs: *mut DBKeyVersions);
+
+    pub fn crocksdb_keyversions_iterator_create(kv: *mut DBKeyVersions)
```

Comment:
btw, the return of crocksdb_get_all_key_versions in c is void, but in rust is size_t.